### ISSUE TITLE
Interrupting the creation of a new key will break the creation of that key afterwards

### ISSUE THREAD

**Issue by vabatta** [TimothyYe/skm#23]
Steps to reproduce the problem:
```
$> skm c my-key
Generating public/private rsa key pair.
Enter passphrase (empty for no passphrase): _
<CTRL+C to interrupt process>
$>
$> skm c my-key
Create SSH key failed!
$>
```

I would expect to be able to create the key again afterward.

**Comment by TimothyYe** [comment]
@vabatta Thanks, I will try to reproduce it.

**Comment by TimothyYe** [comment]
@vabatta Thanks, this issue is already fixed, please download the latest version V0.5.3
